code/game/gamemodes/game_mode.dm

```diff
@@ -54,6 +55,8 @@
 	if(!Debug2)
 		if(playerC < required_players)
 			return 0
+		if(playerC > max_players)

```

Comment:
if you were doing that it'd be better as ```if(max_players && playerC > max_players)```
this eliminates > 0, and also speeds it up so that the playerC count isn't compared against 0 in the first place.